### PR TITLE
update doc for multi cluster

### PR DESCRIPTION
**Pull request by richardwxn**
Otherwise the remote telemetry and policy svc and endpoints would not be created

### CODE REVIEW

File: content/en/docs/setup/install/multicluster/shared-vpn/index.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ cluster. You can install the component in one of two ways:
     $ istioctl manifest apply \
     --set profile=remote \
     --set values.global.controlPlaneSecurityEnabled=false \
+    --set values.global.createRemoteSvcEndpoints=true \
```
Comment (ericvn) [inline content/en/docs/setup/install/multicluster/shared-vpn/index.md]: Should this value also be added to the `Installation configuration parameters` table?

Comment (richardwxn) [reply content/en/docs/setup/install/multicluster/shared-vpn/index.md]: oh yes, I just updated it, also added to some other places

File: content/en/docs/setup/install/multicluster/shared-vpn/index.md [hunk 2]
```diff
@@ -122,6 +123,7 @@ The following table shows the `istioctl` configuration values for remote cluster
 | `values.global.remoteTelemetryAddress` | A valid IP address or hostname | None | Specifies the Istio control plane's telemetry Pod IP address or remote cluster DNS resolvable hostname |
 | `values.sidecarInjectorWebhook.enabled` | true, false | true | Specifies whether to enable automatic sidecar injection on the remote cluster |
 | `values.global.remotePilotCreateSvcEndpoint` | true, false | false | If set, a selector-less service and endpoint for `istio-pilot` are created with the `remotePilotAddress` IP, which ensures the `istio-pilot.<namespace>` is DNS resolvable in the remote cluster. |
+| `values.global.createRemoteSvcEndpoints` | true, false | false | If set, a selector-less services and endpoints for `istio-pilot`, `istio-telemetry`, `istio-policy` are created with the corresponding remote IPs: `remotePilotAddress`, `remoteTelmetryAddress`, `remotePolicyAddress`, which ensures the service names are DNS resolvable in the remote cluster. |
```
Comment (frankbu) [inline content/en/docs/setup/install/multicluster/shared-vpn/index.md]:
```suggestion
| `values.global.createRemoteSvcEndpoints` | true, false | false | If set, selector-less services and endpoints for `istio-pilot`, `istio-telemetry`, `istio-policy` are created with the corresponding remote IPs: `remotePilotAddress`, `remoteTelmetryAddress`, `remotePolicyAddress`, which ensures the service names are DNS resolvable in the remote cluster. |
```

**Comment by frankbu** [comment]
@richardwxn Does this fix need to be cherry-picked to release-1.4?

**Comment by richardwxn** [comment]
@frankbu yep I think so, added the label

**Pull request merged**